send.go

```diff
@@ -99,6 +99,9 @@ func (c *Connection) RoundTrip(request *http.Request) (response *http.Response,
 	}
 
 	// Check that the response content type is JSON:
+	if response.StatusCode == http.StatusNoContent {
+		return
+	}
```

Comment:
IMO this would be better suited in the CheckContentType function itself. It is also used in the transport wrapper and would prevent any future headache if it is used any where else in the future.

Reply:
Moved